src/Mod/Draft/DraftEdit.py

```diff
@@ -228,6 +228,7 @@ def __init__(self):
 
     def GetResources(self):
         return {'Pixmap'  : 'Draft_Edit',
+                'Accel' : "D, E",
```

Comment:
No space before the colon.
```py
'Pixmap': 'Draft_Edit',
'Accel': "D, E",
```

Reply:
@vocx-fc ok!
@yorikvanhavre I tested this shortcut after committing the change. I always test things, but this time i didn't. And it doesn't work (dont produce any errors, just dont work). I guess because _class Edit()_ in not based on DraftTools.Modifier. And if i state _class Edit(DraftTools.Modifier)_ it result in an error.

Reply:
Maybe it works, but it is being over written by another accelerator defined elsewhere in FreeCAD.

Reply:
Yes, your code looks OK... possibly the D letter is used somewhere as a single-letter shortcuts (something we try to remove everywhere, except for very very generic cases ex. view shortcuts 1,2,3...)

Reply:
A hint: i realized the shortcut works in Arch Module but not in Draft...